bitcoinspv/bootstrap.go

```diff
@@ -160,19 +160,19 @@ func (r *Relayer) initializeBTCCache() error {
 // waitForBTCToSyncWithNative ensures BTC node is synchronized by checking
 // that its chain height is at least equal to the Native light client height.
 // This synchronization is required before proceeding with relayer operations.
-func (r *Relayer) waitForBTCToSyncWithNative() error {
+func (r *Relayer) waitForBTCToSyncWithNative(ctx context.Context) error {
```

Comment:
```suggestion
func (r *Relayer) waitForBTCToSyncWithBTCLightClientonSui(ctx context.Context) error {
```

Reply:
This name is so long. I'm feel free if you have other suggestions.

Reply:
renamed